django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate(using, 'auth', Permission):
```

Comment:
I've used `Model._meta.app_label` instead, I'm not too sure about `app_config` since we have a special condition to check if it has modules and `apps.get_model()` hardcodes the string.